.todir

```diff
@@ -1 +1 @@
-# Add bower installer inside dockerfile
+# Add syntax highlighting
```

Comment:
The committing of `.todir` files has been "deprecated" as per previous review.. Should we open a GitHub Issue with the Syntax Highlighting since it seems like a useful feature?